### DISCUSSION TITLE
Why is there a big difference between results of DIANN1.8 and DIANN1.8.1?

### DISCUSSION THREAD

**Discussion by yanhao12355** [vdemichev/DiaNN#722] [Q&A]
Dear author! We use same raw datas , library and almost the same parameters.DIANN1.8  identifies 3422 proteins, but DIANN1.8.1  only identifies 2390 proteins.①We want to know why is there a big difference between results of DIANN1.8 and DIANN1.8.1? ②What's the difference between the places marked in the pictures？Parameter pictures are attached. @vdemichev
![7d74d0292c25946bfc5c7f560505370a](https://github.com/vdemichev/DiaNN/assets/131587580/97f4036f-38e1-4a3d-ad4f-4eb225cc1585)
![aa0d8680c2ae5eecfe7d43e76b3d50c4](https://github.com/vdemichev/DiaNN/assets/131587580/a5651810-35d2-4e78-bb41-1db35973154f)

**Comment by vdemichev** [comment]
This is a distortion of the GUI due to some problem with system fonts. The 'heuristic' option was added and 'remove likely interferences' renamed to 'no shared spectra'.

**Reply by yanhao12355** [reply]
Thank you very much!

**Reply by yanhao12355** [reply]
@vdemichev Dear author, We again find surprise result: 'no shared spectra' is theoretically checked in DIANN1.8.1. but whether  'heuristic' option was checked or not , DIANN1.8.1 could only identify 2390 proteins.Whether 'remove likely interferences' was checked or not, DIANN1.8 identified more proteins(3422 and 3860 respectively).Why is there a big difference between results of DIANN1.8 and DIANN1.8.1?
